README.md

```diff
@@ -186,6 +186,21 @@ configuration](docs/user_guide/model_configuration.md) for the model.
 - Send requests directly to Triton with the [HTTP/REST JSON-based
   or gRPC protocols](docs/customization_guide/inference_protocols.md#httprest-and-grpc-protocols)
 
+#### Backend Support Matrix
+This is a list of all current Nvidia supported Backends, this does not include Third Party or demonstration backends.
+
+| Backend      | Link                                                                   | Status         | Usage Example                                                                                                |
+|--------------|------------------------------------------------------------------------|----------------|--------------------------------------------------------------------------------------------------------------|
+| Python       | [link](https://github.com/triton-inference-server/python_backend)      | 🟢 Recommended | Todo                                                                                                          |
+| Pytorch      | [link](https://github.com/triton-inference-server/pytorch_backend)     | 🟡 Ready       | [tutorial](https://github.com/triton-inference-server/tutorials/blob/main/Quick_Deploy/PyTorch/README.md)    |
+| Tensorflow   | [link](https://github.com/triton-inference-server/tensorflow_backend)  | 🟡 Ready       | [tutorial](https://github.com/triton-inference-server/tutorials/blob/main/Quick_Deploy/TensorFlow/README.md) |
+| ONNX         | [link](https://github.com/triton-inference-server/onnxruntime_backend) | 🟢 Recommended | [tutorial](https://github.com/triton-inference-server/tutorials/blob/main/Quick_Deploy/ONNX/README.md)       |
+| vLLM         | [link](https://github.com/triton-inference-server/vllm_backend)        | 🟡 Ready        | [tutorial](https://github.com/triton-inference-server/tutorials/blob/main/Quick_Deploy/vLLM/README.md)       |
+| TensorRT     | [link](https://github.com/triton-inference-server/tensorrt_backend)    | 🟢 Recommended | [tutorial](https://github.com/NVIDIA/TensorRT/tree/main/quickstart/deploy_to_triton)                         |
+| TensorRT-LLM | [link](https://github.com/triton-inference-server/tensorrtllm_backend) | 🟢 Recommended | Todo                                                                                                         |
```

Comment:
CC @nealvaidya @tanayvarshney if either of you have any good ideas for the missing examples/tutorials in this table. Doesn't have to be strictly from the "tutorials" repo, but some kind of good starter reference for the backend.

Reply:
Sure, here's a few:

[TensorRT-LLM](https://github.com/triton-inference-server/tutorials/blob/main/Popular_Models_Guide/Llama2/trtllm_guide.md)

[FIL](https://github.com/triton-inference-server/fil_backend/tree/main/notebooks/categorical-fraud-detection#fraud-detection-with-categorical-xgboost)

Python backend is a little tricky since there's a lot of different usecases, but one example might be this one: https://github.com/triton-inference-server/tutorials/tree/main/Quick_Deploy/HuggingFaceTransformers

AFAIK we don't have a dedicated OpenVINO example, at least one that I'm aware of.

Reply:
Nice I've added these to the matrix